### PR TITLE
Fix: Add missing configurations

### PR DESCRIPTION
**Pull request by lizable**
## Description
This PR updates multiple configuration files to the latest, which contains every configuration used for a certain purpose corresponding to its name.

Related PR: #1341.

### CODE REVIEW

**Review by adrysn — APPROVED** [review]
LGTM.

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM

**Review by achimnol — CHANGES_REQUESTED** [review]
Shouldn't we update README as well?

File: config.toml.sample [hunk 1]
```diff
@@ -1,54 +1,55 @@
...
-maxCUDASharesPerContainer = 8  # Maximum CUDA shares per container.
-maxShmPerContainer = 4 # Maximum shared memory per container.
-maxFileUploadSize = 4294967296 # Maximum size of single file upload. Set to -1 for unlimited upload.
+openPortToPublic = true          # Show option to open app proxy port to anyone.
+maxCPUCoresPerContainer = 256    # Maximum CPU per container.
+maxMemoryPerContainer = 64       # Maximum memory per container.
+maxCUDADevicesPerContainer = 16  # Maximum CUDA devices per container.
+maxCUDASharesPerContainer = 8    # Maximum CUDA shares per container.
+maxShmPerContainer = 1           # Maximum shared memory per container.
+maxFileUploadSize = 4294967296   # Maximum size of single file upload. Set to -1 for unlimited upload.
```
Comment (achimnol) [inline config.toml.sample]: Are the changes in the default sample values for `openPortToPublic`, `maxShmPerContainer` okay?

Comment (lizable) [reply config.toml.sample]: I just set the same values in the resources section on `debug.toml`, `default.toml` to sync with `config.toml.sample`. 
And I think it's okay, but better to ask @inureyes or @adrysn about this.

Comment (adrysn) [reply config.toml.sample]: Yes. `openPortToPublic` should be `false` by default. I also think the sane default for `maxCUDA*PerContainer` would be `8`.

Comment (lizable) [reply config.toml.sample]: Got it! I'll update those files right away.

**Comment by lizable** [comment]
## IMPORTANT! ##
configuration files in `configs` directory are auto-created via Makefile. You don't need to change the value of each key manually.

File: config.toml.sample [hunk 2]
```diff
@@ -1,46 +1,55 @@
...
 [plugin]
 # Reserved to load plugins
 #login = "signup-cloud.js"
 #sidebar = "report-cloud.js"
+#page = "test-plugin1,test-plugin2" # Show menus on the list which are separated by comma.
 
-
-
+#[license]
+#This field is generated by console-server.
```
Comment (inureyes) [inline config.toml.sample]: Let's modify `console-server `→ `backend.ai-webserver`

**Review by inureyes — CHANGES_REQUESTED** [review]

**Review by inureyes — APPROVED** [review]
LGTM

**Review by inureyes — APPROVED** [review]
LGTM (with small modification)

**Pull request merged**